src/nut_configurator.cc

```diff
@@ -87,12 +87,14 @@ static bool isEpdu(const fty::nut::DeviceConfiguration &config)
 
 static bool isAts(const fty::nut::DeviceConfiguration &config)
 {
+    static const std::set<std::string> atsMibs = {
+        { "eaton_ats16" }, { "eaton_ats16_g2" }, { "eaton_ats30" }, { "apc_ats" }
```

Comment:
just one very-very picky note.
{ "eaton_ats16" }  those {} are uneccessary.

Reply:
Part of the crap, I just copy/pasted for coherence. But this whole code will be trashed once we call nut-driver data dump to get device.type and get rid of such hacky approach. Still, thx for commenting 😉